src/components/Reports/GoalCalculator/GoalCard/GoalCard.test.tsx

```diff
@@ -4,14 +4,37 @@ import { render, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import TestRouter from '__tests__/util/TestRouter';
 import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { BudgetFamilyCategoryEnum } from 'src/graphql/types.generated';
 import theme from 'src/theme';
 import { ListGoalCalculationFragment } from '../GoalsList/GoalCalculations.generated';
 import { GoalCard } from './GoalCard';
+import { useGoalCardCalculations } from './useGoalCardCalculations';
+
+jest.mock('./useGoalCardCalculations');
+const mockUseGoalCardCalculations = jest.mocked(useGoalCardCalculations);
 
 const goal: ListGoalCalculationFragment = {
```

Comment:
Since this mock is getting bigger you could generate it with `gqlMock` and only have to provide a few fields. Search the codebase for how to use `gqlMock` to get mock data for a fragment if you haven't done that before.